app/javascript/application.js

```diff
@@ -8,6 +8,8 @@ import BaseCheckbox from './components/base-checkbox.vue';
 import UserSession from './components/user-session.vue';
 import ItemList from './components/item-list.vue';
 import PurchaseList from './components/purchase-list.vue';
+import ItemDetails from './components/item-details.vue';
+import InlineSvg from 'vue-inline-svg';
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Absolute imports should come before relative imports  import/imports-first